po/R-nl.po

```diff
@@ -0,0 +1,339 @@
+msgid ""
+msgstr ""
+"Project-Id-Version: pkgdown 2.0.9.9000\n"
+"POT-Creation-Date: 2024-05-24 09:36-0500\n"
+"PO-Revision-Date: 2025-10-16 23:28+1\n"
+"Last-Translator: Pepijn de Vries <pepijn.devries@outlook.com>\n"
+"Language-Team: @pepijn-devries\n"
+"Language: nl\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+
+#: build-404.R:11
+msgid "Page not found (404)"
+msgstr "Pagina niet gevonden (404)"
+
+#: build-404.R:12
+msgid "Content not found. Please use links in the navbar."
+msgstr "Inhoud niet gevonden. Gebruik aub de links in de navigatiebalk"
+
+#: build-articles.R:432 navbar.R:186 navbar.R:196 navbar.R:217
+msgid "Articles"
+msgstr "Artikelen"
+
+#: build-articles.R:570
+msgid "All vignettes"
+msgstr "Alle artikelen"
```

Comment:
```suggestion
msgstr "Alle vignetten"
```
Not sure if the distinction is very meaningful, but probably 'vignetten' is a more literal translation